cmake.deps/deps.txt

```diff
@@ -59,3 +59,6 @@ TREESITTER_MARKDOWN_URL https://github.com/MDeiml/tree-sitter-markdown/archive/v
 TREESITTER_MARKDOWN_SHA256 4909d6023643f1afc3ab219585d4035b7403f3a17849782ab803c5f73c8a31d5
 TREESITTER_URL https://github.com/tree-sitter/tree-sitter/archive/v0.22.5.tar.gz
 TREESITTER_SHA256 6bc22ca7e0f81d77773462d922cf40b44bfd090d92abac75cb37dbae516c2417
+
+WASMTIME_URL https://github.com/bytecodealliance/wasmtime/archive/1fa8de140fce161ed5d77cf939558de79d3956c1.tar.gz
```

Comment:
TODO(dundargoc): this some bullshit, wasmtime don't have the header ´wasm.h´ in their release branches. This means we can't use any of their release branches or release tags with the current setup. Unsure if using a main commit is good for now or we need to ask wasmtime why they're doing this.

Reply:
This has been fixed after the v20.0.0 release. See https://github.com/neovim/neovim/pull/28415#issuecomment-2072017990